common/lib/xmodule/xmodule/js/spec/video/sjson_spec.js

```diff
@@ -30,6 +30,31 @@ function (Sjson) {
             expect(sjson.search(21620)).toEqual(4);
             expect(sjson.search(24920)).toEqual(5);
         });
+
+        it('search returns -1 for a value outside the bounds of the array', function() {
+            expect(sjson.search(10024920)).toEqual(sjson.getCaptions().length - 1);
```

Comment:
I may be missing context, but the description and the `expect` seem slightly different.